.eslintrc.json

```diff
@@ -19,7 +19,7 @@
     "import/no-unresolved": "off",
     "no-shadow": "off",
     "import/extensions": "off",
-    "react/forbid-prop-types": "off"
-    
+    "react/forbid-prop-types": "off",
+    "prefer-destructuring": "off"
   }
 }
```

Comment:
- Please do not modify the config files, this file should look like [THIS](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) 
![image](https://user-images.githubusercontent.com/46115725/72563886-35e84e00-3874-11ea-9451-f065819d44b9.png)
- Please refer to [the linter repo here](https://github.com/microverseinc/linters-config/tree/master/react-redux) and follow the instructions to set up your linter, just make sure to select Yes when it prompts to install dependencies. From what I can see there is no eslint installed as a developer dependency in `package.json` file
![image](https://user-images.githubusercontent.com/46115725/72564060-95465e00-3874-11ea-98af-b492cdf91ec2.png)